Common/UnitDefinitions/CloudCover.json

```diff
@@ -0,0 +1,43 @@
+{
+  "Name": "CloudCover",
+  "BaseUnit": "Fraction",
```

Comment:
I know I suggested `Fraction`, but let's await the decision in #880 on whether we want to use `DecimalFraction` or `Decimal` here.

Reply:
OK, is this the only thing left? Everything else LGTY?

Reply:
Sorry yes, I bumped the conversion in #880 to try to move this along.